solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/Service/Entities/ActivityTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FinanceAppTests.swift
+//  FinanceAppTests
+//
+//  Created by Rodrigo Borges on 30/12/21.
+//
+
+import XCTest
+@testable import FinanceApp
+
+class ActivityTests: XCTestCase {
```

Comment:
[sugestao] bem minor, mas uma boa pratica e colocar sempre class como `final`.
```suggestion
final class ActivityTests: XCTestCase {
```

Reply:
Okay, vou ajustar